ngrinder-controller/src/main/java/org/ngrinder/perftest/controller/PerfTestController.java

```diff
@@ -785,8 +744,8 @@ public boolean apply(@Nullable FileEntry input) {
 	 * @return json message containing test info.
 	 */
 	@RestAPI
-	@RequestMapping(value = "/api/{id}", method = RequestMethod.GET)
-	public HttpEntity<String> getOne(User user, @PathVariable("id") Long id) {
+	@GetMapping(value = "/api/{id}")
+	public HttpEntity<String> getOne(User user, @PathVariable Long id) {
```

Comment:
That method should be moved to ```PerfTestApiController```

Reply:
Yes, currently I moved only modified method to PerftestApiController.
this seems like i missed. 

I will clean up all methods at the end of the perftest related implementation.

Reply:
It's great.